docs/source/reading_plans.rst

```diff
@@ -1,123 +1,237 @@
+selecting a plan, you will be shown the Daily Reading screen for the first day
+of the reading plan's schedule.
 
-On first access you select a plan and are taken to the Daily Reading screen for the first day.
+Clicking the `Done` button will take you to the next day's reading schedule.
+Note: The `Done` button will only be enabled after pressing either the `Speak`
+or `Passage` buttons for each passage that is listed. Alternatively, you can tap
+the check next to each passage to enable the `Done` button (manually checking
+the check marks can be useful if you're reading in a paper Bible).
 
-The Done button will be enabled after pressing Speak or Passage for each passage, or after you've tapped each checkbox (which you might use if you're reading in a paper Bible). "Done" marks the current day as complete and if you are behind will take you to the next day's readings.
+|reading_plan_daily_readings| |reading_plan_list| |reading_plan_days|
 
+.. |reading_plan_list| image:: /images/reading_plan_daily_readings.png
+   :width: 32%
 
-How to create a reading plan
+.. |reading_plan_days| image:: /images/reading_plan_days.png
+   :width: 32%
+
+.. |reading_plan_daily_readings| image:: /images/reading_plan_list.png
+   :width: 32%
+
+Abandon Reading Plan
+--------------------
+
+Press the vertical 3-dot menu from the `Daily Reading` screen, and click `Reset`.
+Next time you return to the same `Reading Plan`, the progress will be reset to day 1.
+
+Switch Between Reading Plans
 ----------------------------
 
+From the `Daily Reading` screen, click the first button in the top lef of the
+toolbar. This will let you select a different reading plan. You can return to
+the original reading plan and your progress will remain where you left it.
+In other words, it is possible to use several reading plans simultaneously.
+
+Preview Reading Plans or Go To Specific Day
+-------------------------------------------
+
+When in the `Daily Reading` screen, click the second button in the top left of
+the toolbar. You will see a list of daily readings. Clicking on a daily reading
+will bring you to `Daily Reading` screen for that specific day.
+
+Changing Reading Plan Start Date
+--------------------------------
+
+If you started the readings on a previous day and have just switched to |app|,
+then select the vertical 3-dot menu from the `Daily Reading` screen and
+click `Set Start Date...` to pick a specific start date or click `Set as current day...`
+to set the start date of the reading plan to today's date.
+
+Set Specific Reading Plan Day to Today's Date
+---------------------------------------------
+
+This is useful if you have been reading outside of |app|. To
+move the current reading plan day to today's date:
+
+#. Open the `Daily Reading` screen for the reading plan you would like to adjust
+#. Go to the specific day of the reading plan you want to set to today's date.
+#. Click the top right 3-dot menu
+#. Click `Set as current day`.
```

Comment:
Seems kinda duplicate from previous section..? Maybe could change the title of previous section to something like `Changing Reading Plan Start Date or Progress`